pages/index.js

```diff
@@ -2,9 +2,10 @@ import Head from 'next/head';
 import Image from 'next/image';
 import styles from '../styles/Home.module.css';
```

Comment:
You can (and should) delete the styles folder and all imports that come with it :)

Reply:
Done :)